resources/ory/charts/hydra/charts/hydra-maester/values.yaml

```diff
@@ -50,6 +61,11 @@ deployment:
   # Configure node tolerations.
   tolerations: []
   annotations: {}
+  podLabels: {}
+
+#TODO: Remove - job is removed
```

Comment:
i don't understand what is going on here o.o

Reply:
One leftover from sync-ing :) Removed.